docs/blog/posts/2023/sr-linux-kubernetes-anycast-lab.md

```diff
@@ -0,0 +1,955 @@
+
+Wouldn't it be great to combine both worlds?
+
+In this blog post we will explore a lab topology consisting of a Leaf/Spine [SR Linux](https://learn.srlinux.dev/) Fabric connected to a Kubernetes Cluster.
+
+Our k8s Cluster will feature [MetalLB](https://metallb.universe.tf/), which is a load-balancer implementation for bare metal clusters. This will unlock the possibility to have anycast services in our fabric.
+
+<!-- more -->
+
+## Lab summary
+
+| Summary                   |                                                                                                                                                                                                |
+| ------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Lab name**              | SR Linux Kubernetes Anycast Lab                                                                                                                                                                        |
+| **Lab components**        | Nokia SR Linux, Kubernetes, MetalLB                                                                                                                                                            |
+| **Resource requirements** | :fontawesome-solid-microchip: 6 vCPU <br/>:fontawesome-solid-memory: 16 GB                                                                                                                      |
+| **Lab**                   | [srl-labs/srl-k8s-anycast-lab][lab]                                                                                                                                                               |
+| **Version information**   | [`containerlab:0.42.0`](https://containerlab.dev/install/), [`srlinux:23.3.3`](https://github.com/nokia/srlinux-container-image),[`minikube v1.30.1`](https://minikube.sigs.k8s.io/docs/start/)|
+| **Authors**               | Míchel Redondo [:material-linkedin:][rd-linkedin]                                                                                                                                              |
+
+At the end of this blog post you can find a quick summary of the steps ([TL;DR](#tldr)).
+
+## Prerequisites
+
+The lab leverages [Containerlab](https://containerlab.dev/)  to spin up a Leaf/Spine Fabric coupled with [Minikube](https://minikube.sigs.k8s.io/) to deploy the Kubernetes cluster.
+
+[Docker engine](https://docs.docker.com/engine/install/) has to be installed on the host system.
+
+## Lab topology
+
+The goal of this lab is to provide users with an environment to test the network integration of a Kubernetes cluster with a Leaf/Spine SR Linux fabric.   
+
+
+<center markdown>![topology](sr-linux-kubernetes-anycast-lab/topology.svg){width=100%}</p>
```

Comment:
Another drawings rule - we use Google Sans for text and Fira Code for monospaced text
You can set them in this window in drawio

<img width="476" alt="image" src="https://github.com/srl-labs/learn-srlinux/assets/5679861/ff165017-4867-4c10-bf64-741c5063bae8">